Cargo.toml

```diff
@@ -22,6 +22,7 @@ default = ["linking", "date_time", "pool"]
 date_time = ["rsfbclient-core/date_time", "chrono"]
 dynamic_loading = ["rsfbclient-native/dynamic_loading"]
 linking = ["rsfbclient-native/linking"]
+embedded = []
```

Comment:
Please, change for a name like 'tests_embedded', or something similar

Reply:
Fixed.